src/components/notification/__tests__/NotificationHelpers.ts

```diff
@@ -0,0 +1,124 @@
+import { AssessmentCategories } from '../../../components/assessment/assessmentShape';
+import * as NotificationHelpers from '../NotificationHelpers';
+import { Notification } from '../notificationShape';
+
+const notification1: Notification = {
```

Comment:
The testing code should also be readable. Dont use 1,2,3, to name your variables. Be explicit e.g. `newNotification` or `missionNotification`